misc/multistream-select/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # 0.9.0 [unreleased]
 
+- Make the `V1Lazy` upgrade strategy more interoperable with `V1`.
```

Comment:
Could you extend this comment? E.g. that the listener side changes are removed and that the `V1` identifier is used instead of `V1Lazy`.